torchaudio/kaldi_io.py

```diff
@@ -0,0 +1,99 @@
+# To use this file, the dependency (https://github.com/vesis84/kaldi-io-for-python)
+# needs to be installed. This is a light wrapper around kaldi_io that returns
+# torch.Tensors.
+import numpy as np
+import torch
+
+
+__all__ = [
+    'read_vec_int_ark',
+    'read_vec_flt_scp',
+    'read_vec_flt_ark',
+    'read_mat_scp',
+    'read_mat_ark',
+]
+
+
+def _default_not_imported_method():
+    raise ImportError('Could not import kaldi_io')
```

Comment:
Add an "Did you install it?" to suggest a solution (since it's probably the fact that they didn't install it).